### PR TITLE
feat(subm): loadgen key generation

### PR DESCRIPTION
**Pull request by dckc**
This is deployed (`yarn deploy`) to https://submit.agoric.app/ and works in at least one case.

`yarn start` should run it locally.

tasks:
 - [x] fix `Error: IAM Service Account Credentials API has not been used...`
 - [x] fix `UnhandledPromiseRejectionWarning` from using async request handlers without catching rejections
 - [x] remove obsolete gentx code

### CODE REVIEW

File: subm/src/subm.js [hunk 1]
```diff
@@ -228,26 +249,27 @@ function makeDiscordBot(guild, authorizedRoles, opts) {
 /**
  * @param {GuildMember} member
  * @param {ReturnType<typeof import('gcs-signed-urls')>} storage
+ * @param { ReturnType<import('./firebaseTool.js').makeFirebaseAdmin> } loadGenAdmin
  */
-function makeTestnetParticipant(member, storage) {
+function makeTestnetParticipant(member, storage, loadGenAdmin) {
   if (!member.user) throw RangeError();
+  const { user } = member;
+  const userID = `${user.username}#${user.discriminator}`;
```
Comment (mhofman) [inline subm/src/subm.js:257]: Doesn't discord allow people to change their `username`?

File: subm/src/subm.js [hunk 2]
```diff
@@ -50,6 +51,7 @@ const Site = freeze({
   badLoginPath: '/loginRefused',
   uploadSlogPath: '/participant/slogForm',
   uploadGenTxPath: '/participant/gentxForm',
+  loadGenKeyPath: '/participant/loadGenKey',
```
Comment (mhofman) [inline subm/src/subm.js:54]: You're arguably more right here with capitalization, but in the `testnet-load-generator` we seem to use `loadgen` all lower case..

**Comment by mhofman** [comment]
> * Error: IAM Service Account Credentials API has not been used...

I'm guessing some things needs to be configured in GCP console?

File: subm/src/subm.js (hunk 1 above)
Comment (dckc) [reply subm/src/subm.js:257]: Perhaps, but a condition of getting testnet points is to not change one's username.

**Comment by dckc** [comment]
> I'm guessing some things needs to be configured in GCP console?

Yes, that seems likely

**Comment by dckc** [comment]
## GCP gods satisfied; it works!

I was able to follow my nose using pointers from these two log messages:

> 2021-08-18T06:29:59.217721Z Error: IAM Service Account Credentials API has not been used in project 545002722617 before or it is disabled. Enable it by visiting https://console.developers.google. ...

> 2021-08-18T20:21:17.638502Z Error: The caller does not have permission; Please refer to https://firebase.google.com/docs/auth/admin/create-custom-tokens for more details on how to use and troubleshoot this feature.

and lo...

image

**Review by mhofman — APPROVED** [review]
I've now confirmed that the command generated can be used to authenticate 🎉

**Pull request merged**